Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rocker/shiny:4.2.0
```

Comment:
Nice with the docker file!

If the Singularity is outdated, you can probably remove the "Singularity" file from the repository as well

Reply:
Removed the Singularity file now also!